ZenPacks/zenoss/LinuxMonitor/objects/objects.xml

```diff
@@ -1935,7 +1935,7 @@
             </object>
         </tomanycont>
     </object>
-    <object id="/zport/dmd/Devices/Server/SSH/Linux/rrdTemplates/FileSystem" module="Products.ZenModel.RRDTemplate" class="RRDTemplate">
+    <object id="/zport/dmd/Devices/Server/SSH/Linux/rrdTemplates/LinuxFileSystem" module="Products.ZenModel.RRDTemplate" class="RRDTemplate">
```

Comment:
I'd recommend leaving the monitoring template named *FileSystem* and instead implement `FileSystem.getRRDTemplateName()` as follows.

```python
    def getRRDTemplateName(self):
        return "FileSystem"
```